src/lib.rs

```diff
@@ -87,20 +82,34 @@ pub mod vc {
     pub use super::iter::*;
 }
 
+const fn default_r() -> usize {
+    // I get an error when I omit the return for some reason?
+    // so thats why the explicit return is needed here
+
+    #[cfg(any(test, miri))]
+    return 4;
+    #[cfg(not(any(test, miri)))]
+    return 8;
+}
+
+// TODO
+// What should the docs say here?
```

Comment:
I would use the same docs as for `CustomVc`, and then add a line to `CustomVc`'s docs saying "this differs from `Vc` in that you can customize the incremental resize chunk size (`R`)"